en/sql/tuning.rst

```diff
@@ -3829,6 +3829,168 @@ The join with the *right_tbl* table was eliminated.
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
 
+.. _view_merge:
+
+View Merging
+============
+
+**View Merging** is an optimization for reducing overhead that occur during the processing of view or inline view. 
+When a query includes a view, there is an overhead of creating a temporary table for that view. 
+And it's impossible to perform index scan on temporary tables, leading to performance degradation.
+But, by using **View Merging** to merge the view with table in main query, performance can be improved because index scan is available.
+
+Queries like Query 1 below, which use inline views, make it easier to understand the content of the query.
+
+.. code-block:: sql
+
+        /* Query 1 */
+        SELECT *
+        FROM (SELECT * FROM athlete WHERE nation_code = 'USA') a,
+        (SELECT * FROM record WHERE medal = 'G') b
+        WHERE a.code = b.athlete_code;
+
+If **View Merging** optimization isn't applied, the inline views a and b are processed in advance and stored in temporary storage, 
+and a join operation is performed based on this data.
+
+In this case, the data stored in temporary storage can't use indexes, resulting in significant performance degradation.
```

Comment:
It is better to say "cannot" instead of "can't."

: "In this case, the data stored in temporary storage cannot use indexes, resulting in significant performance degradation."

Reply:
I modified it.